### ISSUE TITLE
shellcheck parses error if parentheses in let expression

### ISSUE THREAD

**Issue by Kontar-pjc** [koalaman/shellcheck#1349]
#### For bugs
- Rule Id: SC1036
- My shellcheck version: 0.5.0 
- [ ] I read the issue's wiki page, e.g. https://github.com/koalaman/shellcheck/wiki/SC2086
- [ ] I tried on shellcheck.net and verified that this is still a problem on the latest commit

#### Here's a snippet or screenshot that shows the problem:

```sh

#! /bin/bash
file_num=10
all_files=20
let per=(file_num*100)/all_files
echo $per

```

#### Here's what shellcheck currently says:
let per=(file_num*100)/all_files
^-- SC2219: Instead of 'let expr', prefer (( expr )) .
    ^-- SC2034: per appears unused. Verify use (or export if used externally).
        ^-- SC1036: '(' is invalid here. Did you forget to escape it?
        ^-- SC1088: Parsing stopped here. Invalid use of parentheses?

#### Here's what I wanted or expected to see:
Everything runs ok. But it parses failed when parentheses in let expression.
I'm using shellcheck in my Gitlab CI, and I can't modify the code writen by the others.
Be appreciate if you can see this.
